branchinfo.txt

```diff
@@ -3,7 +3,7 @@
 # Each line is expected to be in the format "[Name]=[Value]".
 MAJOR_VERSION=1
 MINOR_VERSION=0
-PATCH_VERSION=0
+PATCH_VERSION=2
```

Comment:
Then we shouldn't want the installer to be dotnet-dev-osx-x64.1.0.2-preview2-buildnumber. @leecow , can you comment on it?